cegs_portal/search/templates/search/v1/partials/_experiment_index.html

```diff
@@ -0,0 +1,91 @@
+{% load static i18n %}
+
+{% block content %}
```

Comment:
This is a template, so it doesn't extend base.html and thus, doesn't have a content or inline_javascript block.